src/com/itschool/session6/homework/challenge01/package-info.java

```diff
@@ -0,0 +1,3 @@
+package com.itschool.session6.homework.challenge01;
+
+// 1. Write a Java program where you create two String objects named stringOne and stringTwo with the same value "OpenAI".
```

Comment:
I don't know what happened with the slides (someone must have erroneously modified them), but the requirement of this was to also print if the two strings refer to the same object. I restored the slides with the original sentences.